src/D2L.CodeStyle.Analyzers/ApiUsage/Serialization/SerializerAttributeAnalyzer.cs

```diff
@@ -18,6 +19,7 @@ internal sealed class SerializerAttributeAnalyzer : DiagnosticAnalyzer {
 			);
 
 		private const string SerializerAttributeFullName = "D2L.LP.Serialization.SerializerAttribute";
+		private const string ISerializerFullName = "D2L.LP.Serialization.ISerializer";

```

Comment:
There is a 2nd interface used, `ITrySerializer` that gets wrapped into an `ITrySerializer`

https://search.d2l.dev/xref/lms/lp/framework/core/D2L.LP/LP/Serialization/Resolvers/Reflection/SerializerActivator.cs?r=9d051287#26

![image](https://user-images.githubusercontent.com/12101120/87743223-6faff800-c7b6-11ea-8ed6-cf6097b5d24e.png)